apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -41,6 +41,8 @@ public final class RequestResponsePopulateMdcFilter implements Filter {
   private static final String REQUEST_START_KEY =
       BfdMDC.computeMDCKey(MDC_PREFIX, RESPONSE_PREFIX, "start_milliseconds");
 
+  private static final String RESPONSE_DURATION_MILLISECONDS = "response_duration_milliseconds";
```

Comment:
lets compute the actual MDC key here like is done for REQUEST_START_KEY and then you can use that for setting the request attribute as well as the KDC

Reply:
done